qiskit_experiments/curve_analysis/base_curve_analysis.py

```diff
@@ -138,8 +138,12 @@ def _default_options(cls) -> Options:
             bounds (Dict[str, Tuple[float, float]]): Boundary of fit parameters.
                 The dictionary is keyed on the fit parameter names and
                 values are the tuples of (min, max) of each parameter.
+            fit_method (str): Fit method that LMFIT minimizer uses.
```

Comment:
Why have this separate to the other lmfit options?

Reply:
This is by design of LMFIT. It defines hierarchy in options. Basically, the minimizer is called with "what minimizer do you want to use" and "what options do you want to pass to the minimizer". This is indeed the same with `fitter` and `fitter_options` in tomography analysis.